subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -51,7 +51,7 @@
 /* LLCP Allocations */
 #if defined(LLCP_TX_CTRL_BUF_QUEUE_ENABLE)
 sys_slist_t tx_buffer_wait_list;
-static uint8_t common_tx_buffer_alloc;
+uint8_t common_tx_buffer_alloc;
```

Comment:
I prefer to leave this static, and create a helper function to access this variable when running unit tests, similar to the function 'ctx_buffers_free' in this file

Reply:
Good idea - so done